src/__tests__/CompareResults/beta/ResultsTable.test.tsx

```diff
@@ -1,27 +1,30 @@
+import type { ReactElement } from 'react';
+
 import ResultsTable from '../../../components/CompareResults/beta/ResultsTable';
 import useProtocolTheme from '../../../theme/protocolTheme';
 import { renderWithRouter } from '../../utils/setupTests';
 import { renderHook, screen } from '../../utils/test-utils';
 
-jest.mock('react-router-dom', () => ({
-  ...jest.requireActual<typeof import('react-router-dom')>('react-router-dom'),
-  useSearchParams: () => [new URLSearchParams({ fakedata: 'true' })],
-}));
+function renderWithRoute(component: ReactElement) {
```

Comment:
I'm not super happy with this name, but it's not bad either, tell me what you think!